.github/workflows/tests_and_release.yml

```
@@ -0,0 +1,68 @@
+name: Validate and Release connector
+on:
+  pull_request:
```

To enable running on `main` after merging.
```suggestion
  push:
    branches: [ main ]
  pull_request:  
```